crates/astria-eyre/src/lib.rs

```diff
@@ -86,43 +89,44 @@ fn write_value(err: &dyn Error, f: &mut core::fmt::Formatter<'_>) -> core::fmt::
     Ok(())
 }
 
-#[cfg(feature = "anyhow_support")]
-pub fn anyhow_to_eyre(anyhow_error: anyhow::Error) -> eyre::Report {
-    let boxed: Box<dyn std::error::Error + Send + Sync> = anyhow_error.into();
-    eyre::eyre!(boxed)
-}
+#[cfg(feature = "anyhow")]
+pub mod anyhow_conversion {
```

Comment:
Don't make this a public module. Just reexport the inner functions at the root (as you already do.